www/class/centreon-clapi/CentreonLDAPContactRelation.class.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/*
+ * Copyright 2005-2021 CENTREON
+ * Centreon is developed by : Julien Mathis and Romain Le Merlus under
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+namespace CentreonClapi;
+
+require_once "centreonObject.class.php";
+require_once "centreonUtils.class.php";
+require_once "centreonTimePeriod.class.php";
+require_once "Centreon/Object/Contact/Contact.php";
+require_once "Centreon/Object/Command/Command.php";
+require_once "Centreon/Object/Relation/Contact/Command/Host.php";
+require_once "Centreon/Object/Relation/Contact/Command/Service.php";
+require_once "centreonLDAP.class.php";
+
+/**
+ * Class representating relation between a contact and his LDAP configuration
+ *
+ * @author sylvestre
+ */
+class CentreonLDAPContactRelation extends CentreonObject
+{
+    private const ORDER_NAME = 0;
+    private const LDAP_PARAMETER_NAME = "ar_name";
+
+    protected int $register;
```

Comment:
Do not use typed property for PR which will be cherry-pick on 21.04.x (PHP 7.3)
```suggestion
    /**
     * @var int
     */
    protected $register;
```